docs/userguide/02_schemas.md

```diff
@@ -0,0 +1,81 @@
+(schemas_chapter)=
+# Schemas in BespokeFit
+
+BespokeFit uses [Pydantic] to "Define how data should be in pure, canonical
+python; validate it with *pydantic*." Pydantic helps BespokeFit provide clear
+error messages when it is configured improperly, and also provides inspection
+and documentation tools that BespokeFit uses extensively.
+
+## Models and fields
+
+Not every class in BespokeFit uses Pydantic; those that do are found mostly in
+the [`schema`] module and can be identified in documentation as a `model`,
+which appears in green before their path and name.
+
+Models are characterized by their fields, which are like attributes in Python
+data classes. Fields can be validated at runtime, both according to their type
+annotations and via custom validator functions. Attempting to set a field to 
+an invalid value will immediately raise a clear error, rather than causing a 
+failure later on when the value is used.
+
+A model's fields, validation machinery, and other Pydantic configuration forms
+its schema. A schema describes every possible configuration of a model and is
+useful for investigating what can be done with a class. Schemas are reproduced
+on a model's API reference, and can be accessed programmatically with the 
+[`schema()`] method.
+
+For example, [`BespokeWorkflowFactory`] is a model. It's schema is available
+on its API page, but can also be accessed on the class itself or any instance:
+
+```python
+from openff.bespokefit.workflows.bespoke import BespokeWorkflowFactory
+
+BespokeWorkflowFactory.schema()
+
+factory = BespokeWorkflowFactory()
+factory.schema()
+```
+
```

Comment:
I don't, because it'll fall out of date, and also because schemas are very long, and there's a link to the API documentation that has the info in the previous paragraph. I'm also yet to implement the ability to include output in my doctest setup.